### ISSUE TITLE
SCSS Not Compiling Because 

### ISSUE THREAD

**Issue by indadam** [olefredrik/FoundationPress#676]
I'm getting the following compiling error when I try compiling the SCSS:

    Error: File to import not found or unreadable: font-awesome.
           Load paths:
             /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/assets/scss
             /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/bower_components/foundation/scss
             /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/bower_components/foundation-sites/scss
             /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/bower_components/foundation-sites
             /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/bower_components/motion-ui
             Sass::Globbing::Importer
            on line 21 of /Applications/AMPPS/www/IND/15ATB001-wordpress-v2.0/assets/scss/foundation.scss
      Use --trace for backtrace.

Font Awesome is correctly installed and looks great. I've re-cloned the theme, but it doesn't seem to make a difference.

Thoughts?

**Comment by indadam** [comment]
Not sure if this helps, but I'm getting the following depreciation errors when I'm running "npm install":

     npm WARN deprecated gulp-minify-css@1.2.3: Please use gulp-cssnano instead.
     npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
     npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0

**Comment by nitrokevin** [comment]
I'm having the same problem, you're not on OS X are you by any chance. Interestingly I was able to install JointWP which is another foundation6 wordpress template

**Comment by indadam** [comment]
@nitrokevin I am on OSX, I also don't have any trouble compiling straight Foundation 6, I haven't tried JointWP, in looking at the Docs, it seemed like I'd need to do quite a bit more work to get the customization I can get out of the box with FoundationPress. I went through and tried to update all the packages that it said were depreciated, but they were up-to-date, so I'm guessing the depreciated packages are in FoundationPress. I know just enough about NPM to be dangerous, so I could very well be doing something wrong...

**Comment by nitrokevin** [comment]
I think you are right, I was able to install an earlier version of foundationpress 6 a few weeks ago but then after downloading the version that had the fix for off canvas it no longer installed

**Comment by indadam** [comment]
So, I've deleted and reinstalled several times, and I've noticed that I'm now not getting any of the bower_components generated when I run the build command. Again, I may be doing something wrong here, I just don't know where to start.

**Comment by indadam** [comment]
I've tried commenting out the: font-awesome line in foundations.scss, and it breaks at the next import line which is for motion-ui. It's seeming to not be able to find any of the installed packages. CodeKit is saying that both font-awesome and motion-ui are installed, and I've found both of their folders and contents in assets/components. I continued to comment out the motion-ui import line, just to see if I could clear the error and work backwards, but it errored at the motion-ui additional includes, so I commented those out and I get this error:

     Error: Base-level rules cannot contain the parent-selector-referencing character '&'.
             on line 49 of /Applications/AMPPS/www/IND/15ATB001-wordpress/assets/scss/modules/_topbar.scss
             from line 73 of /Applications/AMPPS/www/IND/15ATB001-wordpress/assets/scss/foundation.scss
       Use --trace for backtrace.

It's looking like it can't find any of the installed packages, so no matter what I comment out, it won't work. Any and all help would be appreciated.

**Comment by colin-marshall** [comment]
What steps did you take after cloning the repo? Did you run npm install? If so, did it automatically run `bower install && gulp build` after installing the `node_modules`? I just did a fresh clone of the repo, ran `npm install` and everything worked as it should. I'm on OS X as well.

**Comment by indadam** [comment]
@colin-marshall Thanks for replying, I'm following the Quickstart Guide on this GitHub repo, these are the exact commands I'm executing:

     $ cd /Applications/AMPPS/www/IND/
     $ git clone https://github.com/olefredrik/FoundationPress.git 15ATB001-wordpress
     $ cd 15ATB001-wordpress
     $ npm install
     $ npm run build

I assume it ran the `bower install && gulp build command` as it always has in the past when I used FoundationPress with version 5, but I'm not 100%.

**Comment by colin-marshall** [comment]
You can look in the in the log to see if it successfully runs `bower install && gulp build` during the `npm install` postinstall process. If you run `bower install` manually does that get it to work?

**Comment by indadam** [comment]
@colin-marshall you're correct, it does indeed run, but it looks like it didn't create the bower_components folder. I ran `bower install` and it appeared to do nothing. Here's the full log:

	$ cd /Applications/AMPPS/www/IND/
	$ git clone https://github.com/olefredrik/FoundationPress.git 
	Cloning into 'FoundationPress'...
	remote: Counting objects: 5306, done.
	remote: Total 5306 (delta 0), reused 0 (delta 0), pack-reused 5306
	Receiving objects: 100% (5306/5306), 7.82 MiB | 1.17 MiB/s, done.
	Resolving deltas: 100% (2827/2827), done.
	Checking connectivity... done.
	$ cd FoundationPress
	$ npm install
	npm WARN deprecated gulp-minify-css@1.2.3: Please use gulp-cssnano instead.
	npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
	npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
	 
	> phantomjs@1.9.19 install /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-uncss/node_modules/uncss/node_modules/phridge/node_modules/phantomjs
	> node install.js

	PhantomJS not found on PATH
	Download already available at /var/folders/gw/v8ddx3716019jq591t1wvzs40000gn/T/phantomjs/phantomjs-1.9.8-macosx.zip
	Verified checksum of previously downloaded file
	Extracting zip contents
	Removing /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-uncss/node_modules/uncss/node_modules/phridge/node_modules/phantomjs/lib/phantom
	Copying extracted folder /var/folders/gw/v8ddx3716019jq591t1wvzs40000gn/T/phantomjs/phantomjs-1.9.8-macosx.zip-extract-1452289610798/phantomjs-1.9.8-macosx -> /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-uncss/node_modules/uncss/node_modules/phridge/node_modules/phantomjs/lib/phantom
	Writing location.js file
	Done. Phantomjs binary available at /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-uncss/node_modules/uncss/node_modules/phridge/node_modules/phantomjs/lib/phantom/bin/phantomjs
	npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
	 
	> spawn-sync@1.0.15 postinstall /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-sass/node_modules/node-sass/node_modules/cross-spawn/node_modules/spawn-sync
	> node postinstall

	|
	> node-sass@3.4.2 install /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-sass/node_modules/node-sass
	> node scripts/install.js

	Binary downloaded and installed at /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-sass/node_modules/node-sass/vendor/darwin-x64-46/binding.node

	> node-sass@3.4.2 postinstall /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-sass/node_modules/node-sass
	> node scripts/build.js

	` /Applications/AMPPS/www/IND/FoundationPress/node_modules/gulp-sass/node_modules/node-sass/vendor/darwin-x64-46/binding.node ` exists. 
	 testing binary.
	Binary is fine; exiting.
	-
	> fsevents@1.0.6 install /Applications/AMPPS/www/IND/FoundationPress/node_modules/browser-sync/node_modules/chokidar/node_modules/fsevents
	> node-pre-gyp install --fallback-to-build

	[fsevents] Success: "/Applications/AMPPS/www/IND/FoundationPress/node_modules/browser-sync/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v46-darwin-x64/fse.node" is installed via remote

	> foundationpress@2.3.0 postinstall /Applications/AMPPS/www/IND/FoundationPress
	> bower install && gulp build

	bower foundation-sites#~6.1.0   cached git://github.com/zurb/foundation-sites.git#6.1.1
	bower foundation-sites#~6.1.0 validate 6.1.1 against git://github.com/zurb/foundation-sites.git#~6.1.0
	bower motion-ui#~1.1.0          cached git://github.com/zurb/motion-ui.git#1.1.1
	bower motion-ui#~1.1.0        validate 1.1.1 against git://github.com/zurb/motion-ui.git#~1.1.0
	bower fontawesome#~4.4.0        cached git://github.com/FortAwesome/Font-Awesome.git#4.4.0
	bower fontawesome#~4.4.0      validate 4.4.0 against git://github.com/FortAwesome/Font-Awesome.git#~4.4.0
	bower jquery#~2.1.4             cached git://github.com/jquery/jquery-dist.git#2.1.4
	bower jquery#~2.1.4           validate 2.1.4 against git://github.com/jquery/jquery-dist.git#~2.1.4
	bower what-input#~1.1.2         cached git://github.com/ten1seven/what-input.git#1.1.4
	bower what-input#~1.1.2       validate 1.1.4 against git://github.com/ten1seven/what-input.git#~1.1.2
	bower motion-ui#~1.1.0         install motion-ui#1.1.1
	bower fontawesome#~4.4.0       install fontawesome#4.4.0
	bower foundation-sites#~6.1.0  install foundation-sites#6.1.1
	bower jquery#~2.1.4            install jquery#2.1.4
	bower what-input#~1.1.2        install what-input#1.1.4

	motion-ui#1.1.1 assets/components/motion-ui
	└── jquery#2.1.4

	fontawesome#4.4.0 assets/components/fontawesome

	foundation-sites#6.1.1 assets/components/foundation-sites
	├── jquery#2.1.4
	└── what-input#1.1.4

	jquery#2.1.4 assets/components/jquery

	what-input#1.1.4 assets/components/what-input
	[13:47:50] Using gulpfile /Applications/AMPPS/www/IND/FoundationPress/gulpfile.js
	[13:47:50] Starting 'build'...
	[13:47:50] Starting 'copy'...
	[13:47:51] Finished 'copy' after 255 ms
	[13:47:51] Starting 'sass'...
	[13:47:56] Starting 'javascript'...
	[13:47:57] Starting 'lint'...
	[13:47:58] Finished 'lint' after 1.03 s
	[13:47:58] Finished 'javascript' after 2.65 s
	[13:47:59] Finished 'sass' after 8.63 s
	[13:47:59] Finished 'build' after 8.88 s
	colors@1.1.2 node_modules/colors

	merge-stream@1.0.0 node_modules/merge-stream
	└── readable-stream@2.0.5 (string_decoder@0.10.31, isarray@0.0.1, process-nextick-args@1.0.6, inherits@2.0.1, util-deprecate@1.0.2, core-util-is@1.0.2)

	run-sequence@1.1.5 node_modules/run-sequence
	└── chalk@1.1.1 (escape-string-regexp@1.0.4, ansi-styles@2.1.0, supports-color@2.0.0, strip-ansi@3.0.0, has-ansi@2.0.0)

	gulp-concat@2.6.0 node_modules/gulp-concat
	├── concat-with-sourcemaps@1.0.4 (source-map@0.5.3)
	└── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)

	gulp-phpcbf@0.1.2 node_modules/gulp-phpcbf
	└── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)

	gulp-plumber@1.0.1 node_modules/gulp-plumber
	└── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)

	gulp-zip@3.0.2 node_modules/gulp-zip
	├── yazl@2.3.0 (buffer-crc32@0.2.5)
	├── chalk@1.1.1 (escape-string-regexp@1.0.4, ansi-styles@2.1.0, supports-color@2.0.0, has-ansi@2.0.0, strip-ansi@3.0.0)
	├── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)
	└── concat-stream@1.5.1 (inherits@2.0.1, typedarray@0.0.6, readable-stream@2.0.5)

	gulp-sourcemaps@1.6.0 node_modules/gulp-sourcemaps
	├── graceful-fs@4.1.2
	├── convert-source-map@1.1.3
	├── strip-bom@2.0.0 (is-utf8@0.2.1)
	├── vinyl@1.1.0 (clone-stats@0.0.1, replace-ext@0.0.1, clone@1.0.2)
	└── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)

	gulp-flatten@0.2.0 node_modules/gulp-flatten
	└── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)

	gulp-phpcs@1.0.0 node_modules/gulp-phpcs
	├── pluralize@1.2.1
	├── chalk@1.1.1 (escape-string-regexp@1.0.4, ansi-styles@2.1.0, supports-color@2.0.0, has-ansi@2.0.0, strip-ansi@3.0.0)
	└── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)

	yargs@3.31.0 node_modules/yargs
	├── camelcase@2.0.1
	├── window-size@0.1.4
	├── y18n@3.2.0
	├── decamelize@1.1.2 (escape-string-regexp@1.0.4)
	├── string-width@1.0.1 (strip-ansi@3.0.0, is-fullwidth-code-point@1.0.0, code-point-at@1.0.0)
	├── os-locale@1.4.0 (lcid@1.0.0)
	└── cliui@3.1.0 (wrap-ansi@1.0.0, strip-ansi@3.0.0)

	gulp-load-plugins@1.2.0 node_modules/gulp-load-plugins
	├── multimatch@2.0.0 (array-differ@1.0.0, array-union@1.0.1, minimatch@2.0.10)
	├── findup-sync@0.2.1 (glob@4.3.5)
	└── resolve@1.1.6

	gulp-util@3.0.7 node_modules/gulp-util
	├── array-differ@1.0.0
	├── array-uniq@1.0.2
	├── object-assign@3.0.0
	├── lodash._reescape@3.0.0
	├── lodash._reevaluate@3.0.0
	├── beeper@1.1.0
	├── fancy-log@1.1.0
	├── lodash._reinterpolate@3.0.0
	├── replace-ext@0.0.1
	├── has-gulplog@0.1.0 (sparkles@1.0.0)
	├── minimist@1.2.0
	├── vinyl@0.5.3 (clone-stats@0.0.1, clone@1.0.2)
	├── chalk@1.1.1 (escape-string-regexp@1.0.4, supports-color@2.0.0, ansi-styles@2.1.0, has-ansi@2.0.0, strip-ansi@3.0.0)
	├── gulplog@1.0.0 (glogg@1.0.0)
	├── lodash.template@3.6.2 (lodash.escape@3.0.0, lodash._basecopy@3.0.1, lodash._isiterateecall@3.0.9, lodash._basevalues@3.0.0, lodash.restparam@3.6.1, lodash._basetostring@3.0.1, lodash.templatesettings@3.1.0, lodash.keys@3.1.2)
	├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)
	└── multipipe@0.1.2 (duplexer2@0.0.2)

	gulp-autoprefixer@3.1.0 node_modules/gulp-autoprefixer
	├── vinyl-sourcemaps-apply@0.2.1 (source-map@0.5.3)
	├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)
	└── postcss@5.0.14 (js-base64@2.1.9, supports-color@3.1.2, source-map@0.5.3)

	dateformat@1.0.12 node_modules/dateformat
	├── get-stdin@4.0.1
	└── meow@3.7.0 (map-obj@1.0.1, trim-newlines@1.0.0, object-assign@4.0.1, decamelize@1.1.2, camelcase-keys@2.0.0, minimist@1.2.0, loud-rejection@1.2.0, redent@1.0.0, normalize-package-data@2.3.5, read-pkg-up@1.0.1)

	gulp-minify-css@1.2.3 node_modules/gulp-minify-css
	├── object-assign@4.0.1
	├── readable-stream@2.0.5 (string_decoder@0.10.31, isarray@0.0.1, inherits@2.0.1, process-nextick-args@1.0.6, util-deprecate@1.0.2, core-util-is@1.0.2)
	├── vinyl-sourcemaps-apply@0.2.1 (source-map@0.5.3)
	├── vinyl-bufferstream@1.0.1 (bufferstreams@1.0.1)
	└── clean-css@3.4.9 (commander@2.8.1, source-map@0.4.4)

	gulp-uglify@1.5.1 node_modules/gulp-uglify
	├── uglify-save-license@0.4.1
	├── deap@1.0.0
	├── isobject@2.0.0 (isarray@0.0.1)
	├── vinyl-sourcemaps-apply@0.2.1 (source-map@0.5.3)
	├── fancy-log@1.1.0 (chalk@1.1.1)
	├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)
	└── uglify-js@2.6.0 (async@0.2.10, uglify-to-browserify@1.0.2, source-map@0.5.3, yargs@3.10.0)

	gulp@3.9.0 node_modules/gulp
	├── interpret@0.6.6
	├── pretty-hrtime@1.0.1
	├── deprecated@0.0.1
	├── archy@1.0.0
	├── minimist@1.2.0
	├── tildify@1.1.2 (os-homedir@1.0.1)
	├── v8flags@2.0.11 (user-home@1.1.1)
	├── semver@4.3.6
	├── chalk@1.1.1 (escape-string-regexp@1.0.4, ansi-styles@2.1.0, supports-color@2.0.0, strip-ansi@3.0.0, has-ansi@2.0.0)
	├── orchestrator@0.3.7 (stream-consume@0.1.0, sequencify@0.0.7, end-of-stream@0.1.5)
	├── liftoff@2.2.0 (extend@2.0.1, rechoir@0.6.2, flagged-respawn@0.3.1, findup-sync@0.3.0, resolve@1.1.6)
	└── vinyl-fs@0.3.14 (graceful-fs@3.0.8, strip-bom@1.0.0, defaults@1.0.3, vinyl@0.4.6, through2@0.6.5, mkdirp@0.5.1, glob-stream@3.1.18, glob-watcher@0.0.6)

	motion-ui@1.1.1 node_modules/motion-ui
	└── jquery@2.2.0

	gulp-notify@2.2.0 node_modules/gulp-notify
	├── node.extend@1.1.5 (is@3.1.0)
	├── lodash.template@3.6.2 (lodash._basecopy@3.0.1, lodash.templatesettings@3.1.0, lodash._isiterateecall@3.0.9, lodash.escape@3.0.0, lodash._basevalues@3.0.0, lodash._basetostring@3.0.1, lodash.restparam@3.6.1, lodash._reinterpolate@3.0.0, lodash.keys@3.1.2)
	├── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)
	└── node-notifier@4.4.0 (shellwords@0.1.0, growly@1.2.0, minimist@1.2.0, semver@4.3.6, which@1.2.1, lodash.clonedeep@3.0.2, cli-usage@0.1.2)

	gulp-uncss@1.0.4 node_modules/gulp-uncss
	├── object-assign@4.0.1
	└── uncss@0.12.1 (is-absolute-url@1.0.0, commander@2.6.0, is-html@1.0.0, bluebird@2.9.34, glob@4.4.2, request@2.53.0, css@1.6.0, phridge@1.0.8, lodash@3.3.1)

	gulp-jshint@2.0.0 node_modules/gulp-jshint
	├── minimatch@2.0.10 (brace-expansion@1.1.2)
	├── through2@0.6.5 (xtend@4.0.1, readable-stream@1.0.33)
	├── rcloader@0.1.2 (rcfinder@0.1.8, lodash@2.4.2)
	└── lodash@3.10.1

	jshint@2.8.0 node_modules/jshint
	├── strip-json-comments@1.0.4
	├── exit@0.1.2
	├── console-browserify@1.1.0 (date-now@0.1.4)
	├── shelljs@0.3.0
	├── minimatch@2.0.10 (brace-expansion@1.1.2)
	├── cli@0.6.6 (glob@3.2.11)
	├── htmlparser2@3.8.3 (domelementtype@1.3.0, entities@1.0.0, domhandler@2.3.0, readable-stream@1.1.13, domutils@1.5.1)
	└── lodash@3.7.0

	autoprefixer@6.2.3 node_modules/autoprefixer
	├── normalize-range@0.1.2
	├── num2fraction@1.2.2
	├── postcss-value-parser@3.2.3
	├── browserslist@1.0.1
	├── postcss@5.0.14 (js-base64@2.1.9, supports-color@3.1.2, source-map@0.5.3)
	└── caniuse-db@1.0.30000384

	gulp-sass@2.1.1 node_modules/gulp-sass
	├── object-assign@4.0.1
	├── vinyl-sourcemaps-apply@0.2.1 (source-map@0.5.3)
	├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)
	└── node-sass@3.4.2 (get-stdin@4.0.1, async-foreach@0.1.3, chalk@1.1.1, mkdirp@0.5.1, nan@2.1.0, glob@5.0.15, npmconf@2.1.2, gaze@0.5.2, cross-spawn@2.1.4, meow@3.7.0, request@2.67.0, node-gyp@3.2.1, sass-graph@2.0.1)

	gulp-if@2.0.0 node_modules/gulp-if
	├── gulp-match@1.0.0 (minimatch@3.0.0)
	├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.5)
	└── ternary-stream@2.0.0 (fork-stream@0.0.4, duplexify@3.4.2)

	browser-sync@2.11.0 node_modules/browser-sync
	├── ucfirst@1.0.0
	├── async-each-series@0.1.1
	├── longest@1.0.1
	├── emitter-steward@1.0.0
	├── easy-extender@2.3.2
	├── dev-ip@1.0.1
	├── immutable@3.7.6
	├── ua-parser-js@0.7.10
	├── query-string@2.4.2 (strict-uri-encode@1.1.0)
	├── opn@3.0.3 (object-assign@4.0.1)
	├── browser-sync-client@2.4.1 (fresh@0.3.0, etag@1.7.0)
	├── portscanner@1.0.0 (async@0.1.15)
	├── resp-modifier@5.0.2 (debug@2.2.0, minimatch@2.0.10)
	├── connect@3.4.0 (utils-merge@1.0.0, parseurl@1.3.0, debug@2.2.0, finalhandler@0.4.0)
	├── serve-static@1.10.0 (escape-html@1.0.2, parseurl@1.3.0, send@0.13.0)
	├── eazy-logger@2.1.2 (opt-merger@1.1.1, tfunk@3.0.1)
	├── meow@3.3.0 (object-assign@3.0.0, camelcase-keys@1.0.0, minimist@1.2.0, indent-string@1.2.2)
	├── foxy@11.1.4 (http-proxy@1.12.0, lodash.merge@3.3.2, resp-modifier@4.0.4)
	├── micromatch@2.3.5 (filename-regex@2.0.0, array-unique@0.2.1, is-glob@2.0.1, is-extglob@1.0.0, normalize-path@2.0.1, expand-brackets@0.1.4, lazy-cache@0.2.7, arr-diff@2.0.0, regex-cache@0.4.2, kind-of@3.0.2, extglob@0.3.1, object.omit@2.0.0, parse-glob@3.0.4, braces@1.8.3)
	├── fs-extra@0.26.4 (path-is-absolute@1.0.0, klaw@1.1.3, jsonfile@2.2.3, graceful-fs@4.1.2, rimraf@2.5.0)
	├── socket.io@1.4.3 (has-binary@0.1.7, debug@2.2.0, socket.io-parser@2.2.6, socket.io-adapter@0.4.0, engine.io@1.6.6, socket.io-client@1.4.3)
	├── serve-index@1.7.2 (escape-html@1.0.2, parseurl@1.3.0, batch@0.5.2, http-errors@1.3.1, mime-types@2.1.9, accepts@1.2.13, debug@2.2.0)
	├── bs-recipes@1.0.5
	├── localtunnel@1.8.0 (openurl@1.1.0, debug@2.2.0, yargs@3.29.0, request@2.65.0)
	├── lodash@3.10.1
	├── browser-sync-ui@0.5.16 (connect-history-api-fallback@0.0.5, stream-throttle@0.1.3, weinre@2.0.0-pre-I0Z7U9OV)
	└── chokidar@1.4.1 (path-is-absolute@1.0.0, inherits@2.0.1, async-each@0.1.6, glob-parent@2.0.0, is-glob@2.0.1, is-binary-path@1.0.1, anymatch@1.3.0, readdirp@2.0.0, fsevents@1.0.6)

	bower@1.6.9 node_modules/bower
	$ npm run build

	> foundationpress@2.3.0 build /Applications/AMPPS/www/IND/FoundationPress
	> gulp build

	[13:51:58] Using gulpfile /Applications/AMPPS/www/IND/FoundationPress/gulpfile.js
	[13:51:58] Starting 'build'...
	[13:51:58] Starting 'copy'...
	[13:51:58] Finished 'copy' after 216 ms
	[13:51:58] Starting 'sass'...
	[13:51:59] Starting 'javascript'...
	[13:51:59] Starting 'lint'...
	[13:51:59] Finished 'lint' after 228 ms
	[13:51:59] Finished 'javascript' after 391 ms
	[13:52:00] Finished 'sass' after 1.49 s
	[13:52:00] Finished 'build' after 1.71 s
	$ bower install
	$

**Comment by colin-marshall** [comment]
In FoundationPress, `bower_components` is changed to `assets/components`.

**Comment by indadam** [comment]
@colin-marshall that's what I was thinking, but it looks like FoundationPress can't find them. They are installed correctly according to my compiler, but they aren't being found during the import.

**Comment by colin-marshall** [comment]
@indadam how is FoundationPress not finding them when your build task completes successfully according to your log you just posted?

**Comment by indadam** [comment]
@colin-marshall no idea, that's why I posted the issue. Here's the compiling error I get when I try to compile the SCSS for the clone I posted above:

	Error: File to import not found or unreadable: font-awesome.
	       Load paths:
	         /Applications/AMPPS/www/IND/FoundationPress/assets/scss
	         /Applications/AMPPS/www/IND/FoundationPress/bower_components/foundation/scss
	         /Applications/AMPPS/www/IND/FoundationPress/bower_components/foundation-sites/scss
	         /Applications/AMPPS/www/IND/FoundationPress/bower_components/foundation-sites
	         /Applications/AMPPS/www/IND/FoundationPress/bower_components/motion-ui
	         Sass::Globbing::Importer
	        on line 21 of /Applications/AMPPS/www/IND/FoundationPress/assets/scss/foundation.scss
	  Use --trace for backtrace.

**Comment by colin-marshall** [comment]
How are you compiling the SCSS? The log clearly shows the SCSS gets compiled after a `gulp build`.

**Comment by indadam** [comment]
@colin-marshall I'm using CodeKit (https://incident57.com/codekit/)

**Comment by colin-marshall** [comment]
There is your issue. The Gulp build system includes the proper import paths. If you just run `npm run watch` while you're working on your project everything will build properly. If you want it to work with CodeKit you need to go through the Sass imports and put in the full path.

**Comment by indadam** [comment]
@colin-marshall I guess I'm confused, is the current version of FoundationPress different than both Foundation 6 Sites and the previous version of FoundationPress? I've been using CodeKit for it and several other projects. It handles several compiling in addition to SCSS that I'd prefer not to lose.

**Comment by colin-marshall** [comment]
When FoundationPress switched to Gulp, the Sass import paths are set here: 
https://github.com/olefredrik/FoundationPress/blob/master/gulpfile.js#L25-L29

So unless there's a way to add those import paths in CodeKit, you're going to need to change all the imports in the SCSS to use the full path.

For example:
`@import 'font-awesome';` should be `@import 'assets/components/fontawesome/scss/font-awesome';`

There is really no reason to use CodeKit though, since the FP build process with Gulp does everything CodeKit does, and possibly more.

**Comment by indadam** [comment]
@colin-marshall ok, thanks for the help.

**Comment by nitrokevin** [comment]
I'm not using code kit myself I use npm watch, but after I run npm install npm build, when I go to the site it looks like no css is loaded, earlier builds of foundation press 6 worked fine

**Comment by colin-marshall** [comment]
Do you get errors in the console? Can you please be more specific about what you mean by "no css is loaded?" Do you mean the foundation.css file is empty?

**Comment by LukePettway** [comment]
Is your node up to date? I had a similar problem, I downloaded the latest node, installed, and then it worked.

**Comment by nitrokevin** [comment]
I've tried again and now I'm getting this:

[23:14:28] Using gulpfile /Volumes/Server Services Data/Library/Server/Web/Data/Sites/web.artisan-creative.co.uk/foundation6/wp-content/themes/FoundationPress/gulpfile.js
[23:14:28] Starting 'build'...
[23:14:28] Starting 'copy'...
[23:14:29] Finished 'copy' after 34 ms
[23:14:29] Starting 'sass'...
[23:14:29] Starting 'javascript'...
[23:14:29] Starting 'lint'...
[23:14:29] Finished 'lint' after 298 ms
[23:14:29] gulp-notify: [Sass Error] assets/scss/global/_settings.scss
Error: File to import not found or unreadable: ../../components/foundation-sites/scss/util/util
       Parent style sheet: /Volumes/Server Services Data/Library/Server/Web/Data/Sites/web.artisan-creative.co.uk/foundation6/wp-content/themes/FoundationPress/assets/scss/global/_settings.scss
        on line 42 of assets/scss/global/_settings.scss
>> @import '../../components/foundation-sites/scss/util/util';
   ^

[23:14:29] Finished 'javascript' after 417 ms
[23:14:31] Finished 'sass' after 2.33 s
[23:14:31] Finished 'build' after 2.36 s

**Comment by colin-marshall** [comment]
@nitrokevin does the `assets/components/foundation-sites` directory exist in your project? If not, run `bower install` and try again.

**Comment by nitrokevin** [comment]
Hi colin, no I don't have assets/components/foundation-sites 
I tried running bower install and it threw up an error:

/usr/local/lib/node_modules/bower/node_modules/mkdirp/index.js:89
throw err0;
Error: EACCES: permission denied, mkdir '/Users/artisan/.config/configstore'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:794:18)
    at Function.sync (/usr/local/lib/node_modules/bower/node_modules/mkdirp/index.js:70:13)
    at Object.create.all.get (/usr/local/lib/node_modules/bower/node_modules/configstore/index.js:41:13)
    at Object.Configstore (/usr/local/lib/node_modules/bower/node_modules/configstore/index.js:27:44)
    at readCachedConfig (/usr/local/lib/node_modules/bower/lib/config.js:19:23)
    at defaultConfig (/usr/local/lib/node_modules/bower/lib/config.js:11:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/bower/lib/index.js:16:32)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)

**Comment by colin-marshall** [comment]
Fix npm permissions: https://docs.npmjs.com/getting-started/fixing-npm-permissions

Then `rm -rf node_modules` in the FoundationPress directory and run `npm install` again.

**Comment by nitrokevin** [comment]
I followed option 2 to fix the permissions but its still not working for me

**Comment by nitrokevin** [comment]
Ok I tried on a different Mac and it worked fine, I guess I need to uninstall npm and node and try fresh, thanks for your time

**Comment by colin-marshall** [comment]
No problem, I would recommend installing node/npm with homebrew. I stopped running into problems like the ones you are having when I did that.

**Comment by nitrokevin** [comment]
Hey colin, can I just as your advice relating to this.

I have installed node on 2 other macs using home-brew as suggested and all works properly.

So I removed node (and npm) from the mac I was having problems with and re-installed with home-brew, looked like it installed correctly, but when I try and npm run watch from this mac I get errors, no errors from the others.

the error is

[16:40:17] Starting 'copy'...
stream.js:74
      throw er; // Unhandled stream error in pipe.
      ^

Error: EPERM: operation not permitted, chmod '/Volumes/Server Services Data/Library/Server/Web/Data/Sites/web.artisan-creative.co.uk/cummins/wp-content/themes/FoundationPress/assets/javascript/vendor/motion-ui/README.md'
    at Error (native)

npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/5.4.1_1/bin/node" "/usr/local/bin/npm" "run" "watch"
npm ERR! node v5.4.1
npm ERR! npm  v3.5.3
npm ERR! code ELIFECYCLE
npm ERR! foundationpress@2.3.0 watch: `gulp`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the foundationpress@2.3.0 watch script 'gulp'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the foundationpress package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs foundationpress
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls foundationpress
npm ERR! There is likely additional logging output above.

But I don't think it is a permission problem else it wouldn't work from the other computers?

**Comment by colin-marshall** [comment]
@nitrokevin 
> But I don't think it is a permission problem else it wouldn't work from the other computers?

I think it's actually the opposite, a permissions problem would make sense why it would not work on one computer and would work on others, because permissions can differ between different computers.

Did you use `sudo` at any point in the process?

**Comment by nitrokevin** [comment]
No I didn't use sudo, I installed foundationpress on our server and I am able to npm run watch from 2 computers to this same folder and it works, but when I try and do the same actually on the server it fails.

Your reply gave me the idea of sudo npm run watch from the server, and it worked! 

So yeah you are right! It is permissions

**Comment by colin-marshall** [comment]
Nice! If I'm understanding your setup correctly, it might be because you used a different user account when you installed it to the server from the other computers, and that user account is different than what you are using when you are actually on the server? Thus the permissions are different and `sudo` overrides that?

**Comment by jayhughes** [comment]
Just worth adding my five pence/cense. Updating your node (if you're a mac user) helped me. 

`brew update`

**Comment by cotswold** [comment]
I've had similar issues getting it to work with terminal, fixed by changing some paths.  I noticed installing bower creates a folder called bower-components, yet the paths to these in gulpfile.js are `assets/components` changing these to `bower-components` fixes most of the problems. 2 other path problems are in the foundation and settings scss files, the paths need an extra ../

Are these problems related to OSX El Capitan/bower or something? I'm new to GitHub/scss/bower and suspect I may be doing something wrong to have had these errors in the first place as the FoundationPress master files still seem to have what appear to be the wrong paths to me... I'm sure they're not though as surely they'd have been updated. Any help is most appreciated! Thanks!
